src/redux/actionCreators.js

```diff
@@ -50,6 +50,11 @@ const setUserProfile = (payload) => ({
   payload,
 });
 
+const setUserData = (userId, email, login) => ({
+  type: ACTION_TYPES.SET_USER_DATA,
+  data: { userId, email, login },
```

Comment:
And again, don't forget that action object should contain "type" and "payload" keys.

Reply:
done